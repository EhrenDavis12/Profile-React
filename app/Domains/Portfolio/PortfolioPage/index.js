/*
 * HomePage
 *
 * This is the first thing users see of our App, at the '/' route
 *
 * NOTE: while this component should technically be a stateless functional
 * component (SFC), hot reloading does not currently support SFCs. If hot
 * reloading is not a necessity for you then you can refactor it and remove
 * the linting exception
 */

import React from 'react';

/* eslint-disable react/prefer-stateless-function */
export default class PortfolioPage extends React.PureComponent {
  render() {
    return (
      <React.Fragment>
        <h1>PortfolioPage</h1>
      </React.Fragment>
    );
  }
}
