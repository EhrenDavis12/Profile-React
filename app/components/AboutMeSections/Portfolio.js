// import PropTypes from 'prop-types';
import React from 'react';
import Banner from 'components/Banners/Banner';
import DesertCodeCamp from 'components/Projects/DesertCodeCamp';
import CodeChallenges from 'components/Projects/CodeChallenges';
import BootCampTeamFinal from 'components/Projects/BootCampTeamFinal';
import PortfolioWebSite from 'components/Projects/PortfolioWebSite';
import MicroServiceSample from 'components/Projects/MicroServiceSample';

function ProfessionalIntro() {
  return (
    <div className="container">
      <Banner>
        <DesertCodeCamp />
      </Banner>

      <Banner>
        <CodeChallenges />
      </Banner>

      <Banner>
        <BootCampTeamFinal />
      </Banner>

      <Banner>
        <PortfolioWebSite />
      </Banner>

      <Banner>
        <MicroServiceSample />
      </Banner>

      {/* <Banner>
          <br />
          This is a project I worked on to learn/refine technologies.
          React-redux(-sagas), jest - and more.
          <br />
          <a href="https://github.com/EhrenDavis12/Profile-React">
            GetHub Repository
          </a>
          <br />
          <a href="https://ehren-davis-profile-app.herokuapp.com/">
            Go To Web Site
          </a>
          <br />
          No content on the web site, itâ€™s the code here that is fun.
          <br />
          <br />
          This is a basic microservice I built for emails and storing messages.
          The code here is fun! I like the way errors are handled with the async
          calls.
          <br />
          <a href="https://github.com/EhrenDavis12/contactMeAPI">
            GetHub Repository
          </a>
          <br />
          <br />
          !!! ITS STAR WARS !!!
          <br />
          <a href="https://ehrendavis12.github.io/RPG_StarWars/">
            Go To Web Site
          </a>
          <br />
          <a href="https://github.com/EhrenDavis12/RPG_StarWars">
            GetHub Repository
          </a>
        </Banner> */}
    </div>
  );
}

export default ProfessionalIntro;
