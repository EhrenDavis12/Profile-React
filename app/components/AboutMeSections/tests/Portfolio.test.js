import React from 'react';
import { shallow } from 'enzyme';
import 'jest-styled-components';
import Portfolio from '../Portfolio';

describe('<Portfolio />', () => {
  it('should render the page', () => {
    const renderedComponent = shallow(<Portfolio />);
    expect(renderedComponent).toMatchSnapshot();
  });
});
