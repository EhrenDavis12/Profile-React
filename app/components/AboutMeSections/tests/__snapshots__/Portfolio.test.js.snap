// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Portfolio /> should render the page 1`] = `
<div
  className="container"
>
  <BannerSection>
    <DesertCodeCamp />
  </BannerSection>
  <BannerSection>
    <BootCampTeamFinal />
  </BannerSection>
  <BannerSection>
    <PortfolioWebSite />
  </BannerSection>
  <BannerSection>
    <PortfolioWebSite />
  </BannerSection>
</div>
`;
